nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/java/org/nuxeo/ecm/platform/audit/service/AbstractAuditBackend.java

```diff
@@ -109,7 +110,8 @@
 
     protected final AuditBackendDescriptor config;
 
-    protected final CursorService<Iterator<LogEntry>, LogEntry> cursorService = new CursorService<>();
+    protected final CursorService<Iterator<LogEntry>, LogEntry, LogEntry> cursorService = new CursorService<>(
+            Function.identity());
```

Comment:
FYI I've seen JDK devs actually recommend using `entry -> entry` instead of `Function.identity()`, as it's often clearer to the reader.